### DISCUSSION TITLE
Enable reading list

### DISCUSSION THREAD

**Discussion by giaaaacomo** [zen-browser/desktop#2629] [Ideas]
Firefox has its own reading list that is disabled by default although working on ios (more info to be found [here](https://wiki.mozilla.org/QA/Reading_list) ). It would be really cool to have that feature, and using the embedded one would also sync it with those devices (even if i'm on android, where there isn't that feature, that would be imo the best way to implement that). i don't know how, but it would be extra cool if it was also able to merge with pocket.

**Comment by nickserv** [comment]
Unfortunately, it seems like development of Firefox's reading list was already discontinued (bug [1123529](https://bugzilla.mozilla.org/show_bug.cgi?id=1123529)). However if there's bandwidth to work on it, it would be nice to have a basic reading list sidebar in Zen.

**Comment by axlmsg** [comment]
Some wishful thinking for a reading list feature: A cross-device reading list is really useful (typical scenario: Find a link to a relevant long piece of text on mobile device, add to desktop browser for subsequent detailed reading, bookmarking, referencing, and adding to a relevant reference library such as BibTex/Mendeley/...). On macOS, it would be most attractive if Zen could link to the (Safari) "Reading List" which is inherently cross-device already, and "Add to Reading List" is accessible from many apps through "Share" function. (Maybe this is technically not feasible at all, as it is unclear how accessible the Safari Reading List is on macOS/iOS to non-Safari-Apps)

**Comment by ntwrknrd** [comment]
I agree, having 'Add to Reading List' in the macOS share menu would be ideal. I was excited to discover the `browser.menu.share_url.allow` option in about:config, but then disappointed when I realized Safari-related items were missing.

I created a workaround using Shortcuts, but it's far from elegant:

Right-click URL → Share → Shortcuts → Select Shortcut (from Share Sheet) → Click Share → Accept privacy warning.

The shortcut is simple, maybe someone can improve:

![Screenshot-20250921204431@2x](https://github.com/user-attachments/assets/f8e62617-7d3a-4978-b94e-8c3a6cf46ebb)
